PROJECT: Impact of HIV Infection and Antiretroviral Therapy on the Regulatory Epigenome through Methylome Modifications
(Abigail Jackson and Esther Koh, 2023)

OVERVIEW: 
At large, this project seeks to understand the impact of HIV infection and antiretroviral therapy (ART) on epigenetic regulation of the genome by analyzing the positional enrichment of DMPs and rescued-CpG positions near regulatory landmarks, including enhancers, promoters, CTCF-binding sites, and DNAse-I hypersensitive sites. This repo specifically focuses on a subset of the HIV data collected from "Effect of HIV infection and antiretroviral therapy initiation on genome-wide DNA methylation patterns" (Esteban-Cantos et al., 2023) to generate a figure comparing the degree of hypermethylation on each chromosome across three experimental groups: HIV- donors, HIV+/pre-ART donors, and HIV+/post-ART donors. 

SYSTEM REQUIREMENTS: 
pandas1.5.2, seaborn0.11.2, matplotlib3.6.2, numpy 1.23.4, Python Notebook editor of choice (code originated in VSCode)

CODE AVAILABILITY:
"koh_e_pset4_figure.ipynb" â€“ This Jupyter notebook found in the Code folder generates a figure depicting the amount of hypermethylation on each chromosome across three experimental groups: HIV- donors, HIV+/pre-ART donors, and HIV+/post-ART donors. To run this code, you will need the csv file "CombinedData_data_cpg_alignment.csv " within the Data folder. This input file contains counts for the number of donors with hypermethylation at each CpG site across the three experimental groups. Within your Python Notebook editor, execute all steps in the notebook to generate a histogram containing distributions of number of hypermethylated CpG sites on each chromosome across three experimental groups: HIV+/pre-ART, HIV+/post-ART, and HIV- controls. This graph will be automatically saved within your local repo (also found in the Figures folder).

DATA AVAILABILITY: 
The data used in this analysis includes CpG sites and their genomic positions as generated from the validation of Illumina's DNA methylation microarray. These loci are a common reference standard when working with methylation data generated from Illumina-brand assays. As follows, the experimental methylation data used in this analysis was generated by sampling whole blood from donors across three experimental conditions and using Illumina's Infinium MethylationEPIC assay. The data was normalized by Esteban-Cantos et al. using a ssNoob normalization strategy before being uploaded to the GEO database ("DataTable_GSE217633_preprocessSheet.tsv" from https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE217633 and "CpGPositions_GPL21145-48548.txt" from https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GPL21145).
    

